apps/coscrad-frontend/src/components/resources/photographs/photograph-detail.full-view.presenter.tsx

```diff
@@ -35,7 +33,7 @@ export const PhotographDetailFullViewPresenter = ({
             />
             <ImageFullPageWidth image={image} />
             <ResourceNamePresenter name={name} variant="h2" />
-            <SinglePropertyPresenter display="Photographer" value={photographer} />
+            {/* Add Contributor to replace Photographer */}
```

Comment:
If this is the path we are going down, let's do this and add cypress test coverage.

Reply:
I've reverted the deletion of photographer to keep the flexibility.  In which case, do we need to bring in Contributors for a photograph?  I guess if an Elder provided it, they could be a contributor?

Reply:
You should always support the `contributorIds` on the meta if they are there. At this point, `photographer` is a hack that allows you to bring in external photographers without registering them as contributors.

Reply:
Looks like this is done now.